ckanext/dcatde/harvesters/dcatde_rdf.py

```diff
@@ -114,6 +114,53 @@ def after_parsing(self, rdf_parser, harvest_job):
 
         return rdf_parser, error_messages
 
+    def _assign_owner_org(self, harvest_object, dataset_dict):
+        base_context = {'model': model, 'session': model.Session,
+                'user': self._get_user_name()}
+
+        remote_orgs = json.loads(harvest_object.job.source.config).get('remote_orgs', None)
+        if remote_orgs in ('only_local', 'create'):
+                publisher_name = get_extras_field(dataset_dict, 'publisher_name')
+                if (publisher_name):
+                    publisher_name = " ".join(publisher_name.get('value').split())
+
+                # Look for the publisher by name
+                owner_org = None
+
+                organizations = model.Session.query(model.Group.id).filter(model.Group.state == 'active').filter(model.Group.is_organization == True).filter(model.Group.title == publisher_name).all()
+
+                if len(organizations) == 1:
+                    for org in organizations:
+                        owner_org = org.id
+                if len(organizations) == 0:
+                    # Look for an alternative name contained the extra values of the organization
+                    organizations = model.Session.query(GroupExtra.group_id).\
+                        filter(GroupExtra.key.like('alternate_name%')).\
+                        filter(GroupExtra.value == publisher_name ).all()
+                    if len(organizations) == 1:
+                       for org in organizations:
+                           owner_org = org.group_id
+
+                if not owner_org:
+                    if remote_orgs == 'create':
```

Comment:
Theoretisch könnte es vorkommen, dass "fälschlicherweise" mehrere Organisationen mit demselben Titel existieren. Bei `len(organizations) > 1` , wird dann eine neue Organisation angelegt. In diesem Fall wäre es vielleicht sinnvoll, eine Fehlermeldung im Harvestlog zu hinterlegen oder stattdessen die erste ID aus der `organizations`-Liste zuzuweisen.